Dockerfile

```diff
@@ -20,6 +20,7 @@ COPY Makefile .
 
 COPY go.mod .
 COPY go.sum .
+COPY ./api /go/api
```

Comment:
I had to do this to solve [this issue](https://github.com/cosmos/ibc-go/actions/runs/8256316313/job/22584782319) in the docker build.